core/src/i18n/en.yaml

```
@@ -485,7 +485,7 @@ cron-jobs:
       month: 1 to 12
       starting-deadline: Optional deadline in seconds for starting the job if it misses the scheduled time.
       suspend: 'If true, the controller suspends subsequent executions (default: false). It does not apply to started executions.'
-  description: CronJobs are for regular scheduled actions. A CronJob object is like one line of a crontab file, running a job periodically on a given schedule.
+  description: ' is for regular scheduled actions. A CronJob object is like one line of a crontab file, running a job periodically on a given schedule.'
```

```suggestion
  description: ' is for regular scheduled actions. A CronJob object is like one line of a crontab file, running a job periodically on a given schedule.'
```
I'm not 100% sure about the explanation. It references another concept (`crontab`), which to me is not clear BTW, and thus assumes a certain level of understanding from the user, but if we expect the user to know certain things, then why do we need to explain others? 
How would you explain CronJob to someone who doesn't know a first thing about it in a conversation?

---

Maybe `allows to execute actions on a regular basis`?

---

Also, note that you should not take stuff off another website (Kubernetes) without quoting it, otherwise it might be plagiarism. 
So here, you'd want to use your own words.